app/src/main/java/org/mozilla/fenix/telemetry/TelemetryMiddleware.kt

```diff
@@ -31,11 +30,9 @@ import org.mozilla.fenix.GleanMetrics.EngineTab as EngineMetrics
  *
  * @property settings reference to the application [Settings].
  * @property adsTelemetry reference to [AdsTelemetry] use to record search telemetry.
- * @property metrics reference to the configured [MetricController] to record general page load events.
  */
 class TelemetryMiddleware(
     private val settings: Settings,
-    private val metrics: MetricController
```

Comment:
This is an off-side question. I am wondering why our linter in CI doesn't catch these and fail the build in case this is something that is obvious and easy to fix for anybody.

Reply:
I am not aware of such a rule (unused property) existing in either [detekt](https://detekt.dev/docs/rules/style/#unusedprivatemember) or [ktlint](https://ktlint.github.io/#rules) for this. There is a warning in IDE.

Reply:
I would've imagined unused private member in https://detekt.dev/docs/rules/style/#unusedprivatemember should've caught this.